src/Tribe/Views/V2/View.php

```diff
@@ -1035,33 +1035,34 @@ protected function setup_template_vars() {
-			'next_url'             => $this->next_url( true ),
+			'title'                    => $this->get_title( $events ),
+			'events'                   => $events,
+			'url'                      => $this->get_url( true ),
+			'prev_url'                 => $this->prev_url( true ),
+			'next_url'                 => $this->next_url( true ),
 			'bar'                  => [
 				'keyword' => $this->context->get( 'keyword', '' ),
 				'date'    => $this->context->get( 'event_date', '' ),
 			],
-			'today'                => $this->context->get( 'today', 'today' ),
-			'now'                  => $this->context->get( 'now', 'now' ),
-			'rest_url'             => tribe( Rest_Endpoint::class )->get_url(),
-			'rest_nonce'           => wp_create_nonce( 'wp_rest' ),
-			'should_manage_url'    => $this->should_manage_url,
-			'today_url'            => $this->get_today_url( true ),
-			'prev_label'           => $this->get_link_label( $this->prev_url( false ) ),
-			'next_label'           => $this->get_link_label( $this->next_url( false ) ),
-			'date_formats'         => (object) [
+			'today'                    => $this->context->get( 'today', 'today' ),
+			'now'                      => $this->context->get( 'now', 'now' ),
+			'rest_url'                 => tribe( Rest_Endpoint::class )->get_url(),
+			'rest_nonce'               => wp_create_nonce( 'wp_rest' ),
+			'should_manage_url'        => $this->should_manage_url,
+			'today_url'                => $this->get_today_url( true ),
+			'prev_label'               => $this->get_link_label( $this->prev_url( false ) ),
+			'next_label'               => $this->get_link_label( $this->next_url( false ) ),
+			'date_formats'             => (object) [
 				'compact'        => Dates::datepicker_formats( tribe_get_option( 'datepickerFormat' ) ),
 				'month_and_year' => tribe_get_date_option( 'monthAndYearFormat', 'F Y' ),
 			],
-			'messages'             => $this->get_messages( $events ),
-			'start_of_week'        => get_option( 'start_of_week', 0 ),
-			'breadcrumbs'          => $this->get_breadcrumbs(),
-			'display_events_bar'   => $this->filter_display_events_bar( $this->display_events_bar ),
-			'disable_event_search' => tribe_is_truthy( tribe_get_option( 'tribeDisableTribeBar', false ) ),
-			'live_refresh'         => tribe_is_truthy( tribe_get_option( 'liveFiltersUpdate', true ) ),
+			'messages'                 => $this->get_messages( $events ),
+			'start_of_week'            => get_option( 'start_of_week', 0 ),
+			'breadcrumbs'              => $this->get_breadcrumbs(),
+			'display_events_bar'       => $this->filter_display_events_bar( $this->display_events_bar ),
+			'disable_event_search'     => tribe_is_truthy( tribe_get_option( 'tribeDisableTribeBar', false ) ),
+			'live_refresh'             => tribe_is_truthy( tribe_get_option( 'liveFiltersUpdate', true ) ),
+			'display_recurring_toggle' => tribe_is_truthy( tribe_get_option( 'userToggleSubsequentRecurrences', false ) ),
```

Comment:
This should live in PRO.
In PRO, in the Hooks class, filter the view template vars (there should be something already) and set it there.